### PR TITLE
Fix unlinked IS fields in PROTO

### PR DESCRIPTION
**Pull request by ad-daniel**
**Description**
By making a cleanse and letting webots re-build the PROTO cache for all files two additional warnings surfaced.

### CODE REVIEW

File: docs/guide/appearances.md [hunk 1]
```diff
@@ -1380,6 +1380,8 @@ ReflectiveSurface {
 
 ### ReflectiveSurface Field Summary
 
+- `colorOverride`: Defines the default color multiplied with the texture color.
```
Comment (omichel) [inline docs/guide/appearances.md:1383]: I am sure about the "default"...
```suggestion
- `colorOverride`: Defines the color to be multiplied with the texture color.
```

File: docs/guide/appearances.md [hunk 2]
```diff
@@ -1583,6 +1586,8 @@ Rubber {
 
 - `type`: Defines the rubber type. This field accepts the following values: `"flat"` and `"dotted"`.
 
+- `colorOverride`: Defines the default color multiplied with the texture color.
```
Comment (omichel) [inline docs/guide/appearances.md:1589]:
```suggestion
- `colorOverride`: Defines the color to be multiplied with the texture color.
```

File: projects/appearances/protos/ReflectiveSurface.proto [hunk 3]
```diff
@@ -8,7 +8,7 @@ PROTO ReflectiveSurface [
   field SFFloat metalness         0.5
   field SFColor emissiveColor     0 0 0
   field SFFloat emissiveIntensity 1
-  field SFColor baseColorMap      1 1 1
+  field SFColor colorOverride     1 1 1  # Defines the default color multiplied with the texture color.
```
Comment (omichel) [inline projects/appearances/protos/ReflectiveSurface.proto:11]:
```suggestion
  field SFColor colorOverride     1 1 1  # Defines the color to be multiplied with the texture color.
```

File: projects/appearances/protos/Rubber.proto [hunk 4]
```diff
@@ -7,6 +7,7 @@
 
 PROTO Rubber [
   field SFString{"flat", "dotted"} type              "flat"   # Defines the rubber type.
+  field SFColor                    colorOverride     1 1 1    # Defines the default color multiplied with the texture color.
```
Comment (omichel) [inline projects/appearances/protos/Rubber.proto:10]:
```suggestion
  field SFColor                    colorOverride     1 1 1    # Defines the color to be multiplied with the texture color.
```

**Review by omichel — CHANGES_REQUESTED** [review]

**Comment by ad-daniel** [comment]
Closing this PR since it will be addressed here https://github.com/cyberbotics/webots/pull/4190. Changing object protos requires regenerating the docs, and changes to the docs should target released. Since we don't merge master into released other than when creating a new version, it's better to do the cleanup directly in released.

**Pull request closed without merging**